lmfdb/modular_curves/family.py

```diff
@@ -40,464 +49,157 @@ def title(self):
-    moduli_description = fr"$X_{{\mathrm{{arith}}}}(N)$ is the {display_knowl('modcurve','modular curve')} $X_H$ for $H\leq \GL_2(\widehat{{\Z}})$ the inverse image of $\begin{{pmatrix}}1&0\\0&*\end{{pmatrix}}\leq \GL_2(\Z/N\Z)$. As a moduli space, $X_{{\mathrm{{arith}}}}$ parametrizes isomorphism classes of triples $(E,\phi,P)$, where $E$ is a generalized elliptic curve, $P$ is a point of exact order $N$, and $\phi \colon E \to E'$ is a cyclic $N$-isogeny such that $E[N]$ is generated by $P$ and $\ker\phi$. Alternatively, it parametrizes isomorphism classes of pairs $(E,\psi)$ where $E$ is a generalized elliptic curve and $\psi \colon \mu_N\times \Z/N\Z\xrightarrow{{\sim}} E[N]$ is a symplectic isomorphism."
-    genus_formula = r"$$ g = 1 + \frac{ i }{12} - \frac{ \nu_2 }{4} - \frac{ \nu_3}{3} - \frac{ \nu_\infty}{2}$$"
-    biell_level = "?level=7%2C8&family=Xarith"
-    hypell_description = r'[<a href="https://mathscinet.ams.org/mathscinet/article?mr=3145452">MR:3145452</a>] showed that there are no hyperelliptic curves in this family.'
-    biell_description = r'[<a href="https://mathscinet.ams.org/mathscinet/article?mr=3145452">MR:3145452</a>] showed that there are only 2 bielliptic curves in this family. $X_{\textup{arith}}(7)$ is isomorphic to the Klein quartic and $X_{\textup{arith}}(8)$ is isomorphic to the Wiman curve. See the [<a href="https://beta.lmfdb.org/ModularCurve/Q/?level=7%2C8&family=Xarith">following table</a>] for the full list.'
-
-    @lazy_attribute
-    def cusps_display(self):
-        return self.cusps
-
-    @lazy_attribute
-    def cusps_width_display(self):
-        return "unknown"
-
-    @lazy_attribute
-    def cusps_orbits_display(self):
-        return "one is rational"
-
-    elliptic_points = "There are no elliptic points, unless N=1, in which case there are 2"
-
-    @lazy_attribute
-    def properties(self):
-        return [("Level", "$N$"),
-                (r"$SL_2$-level", str(self.sl2level)),
-                ("Index", str(self.index)),
-                ("Genus", str(self.genus))]
+X0 = ModCurveFamily_base(
+    famname="X0",
+    name="X_0(N)",
+    psl2index=r"i = N\cdot \prod\limits_{p|N, \text{ prime}} \left(1 + \frac{1}{p}\right)",
+    nu2=r"$\nu_2 = \begin{cases} 0, & \text{ if } N \equiv 0 \pmod{4}, \\ \prod\limits_{p|N, \text{prime}} \big( 1 + \big( \frac{-1}{p} \big) \big), & \text{otherwise.} \end{cases}$",
+    nu3=r"$\nu_3 = \begin{cases} 0, & \text{ if } N \equiv 0 \pmod{9}, \\ \prod\limits_{p|N, \text{ prime}} \big( 1 + \big( \frac{-3}{p} \big) \big), & \text{otherwise.} \end{cases}$",
+    cusps=r"$\nu_\infty = \sum\limits_{d|N, d>0} \varphi\left(\left(d,\frac{N}{d}\right)\right)$",
+    knowl_ID="modcurve.x0",
+    genus_formula=r"$$ g = 1 + \frac{ i }{12} - \frac{ \nu_2 }{4} - \frac{ \nu_3}{3} - \frac{ \nu_\infty}{2}$$",
+    #hypell_level="?level=22%2C+23%2C+26%2C+28%2C+29%2C+30%2C+31%2C+33%2C+35%2C+37%2C+39%2C+40%2C+41%2C+46%2C+47%2C+48%2C+50%2C+59%2C+71&family=X0",
```

Comment:
Assuming these knowls have been updated, should probably delete these lines rather than comment them out.